metrics/parse_tfm.py

```diff
@@ -64,11 +64,14 @@ def __init__(self, start_char, end_char, char_info, width_table,
         self.ligkern_program = LigKernProgram(ligkern_table)
         self.kern_table = kern_table
 
-    def get_char_metrics(self, char_num):
+    def get_char_metrics(self, char_num, zero_indexed=False):
         if char_num < self.start_char or char_num > self.end_char:
             raise RuntimeError("Invalid character number")
 
-        info = self.char_info[char_num + self.start_char]
+        if zero_indexed:
+            info = self.char_info[char_num - self.start_char]
+        else:
+            info = self.char_info[char_num + self.start_char]
```

Comment:
I tried implementing the TODO, but I could get it to work.  I'm probably misunderstanding something about how the mapping works.  I'm going to leave this as is and add some comments.